recipe/build.sh

```diff
@@ -1,5 +1,32 @@
 #!/bin/bash
 
-./bootstrap --prefix=$PREFIX
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+#
+source activate "${CONDA_DEFAULT_ENV}"
+
+if [[ `uname` == 'Darwin' ]];
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+
+./bootstrap \
+             --prefix="${PREFIX}" \
+             --system-curl \
+             --system-bzip2 \
+             --system-expat \
+             --system-jsoncpp \
+             --system-libarchive \
+             --system-liblzma \
+             --system-zlib \
+             --no-qt-gui \
```

Comment:
The `CPU_COUNT` value tends to be inaccurate on the CIs. We have explored setting this through `MAKEFLAGS` in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/1 ). However, that turns out to be more trouble than it is worth and removal is thus being proposed in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/18 ). Ideally we will override `CPU_COUNT` so that it can be used responsibly, but we will need to get some other players on-board first.